SCClassLibrary/Common/Control/Server.sc

```diff
@@ -248,40 +249,8 @@ ServerOptions {
 		if (maxLogins.notNil, {
 			o = o ++ " -l " ++ maxLogins;
 		});
-
-		// additions for BELA
-		if (numAnalogInChannels.notNil, {
-			o = o ++ " -J " ++ numAnalogInChannels;
-		});
-		if (numAnalogOutChannels.notNil, {
-			o = o ++ " -K " ++ numAnalogOutChannels;
-		});
-		if (numDigitalChannels.notNil, {
-			o = o ++ " -G " ++ numDigitalChannels;
-		});
-		if (headphoneLevel.notNil, {
-			o = o ++ " -Q " ++ headphoneLevel;
-		});
-		if (pgaGainLeft.notNil, {
-			o = o ++ " -X " ++ pgaGainLeft;
-		});
-		if (pgaGainRight.notNil, {
-			o = o ++ " -Y " ++ pgaGainRight;
-		});
-		if (speakerMuted.notNil, {
-			o = o ++ " -s " ++ speakerMuted;
-		});
-		if (dacLevel.notNil, {
-			o = o ++ " -x " ++ dacLevel;
-		});
-		if (adcLevel.notNil, {
-			o = o ++ " -y " ++ adcLevel;
-		});
-		if (numMultiplexChannels.notNil, {
-			o = o ++ " -g " ++ numMultiplexChannels;
-		});
-		if (belaPRU.notNil, {
-			o = o ++ " -T " ++ belaPRU;
+		if (thisProcess.platform.hasFeature(\_Bela_isSupported), {
```

Comment:
you should change this to a real method returning a boolean, a few name suggestions are `hasBelaSupport`, `isBelaSupported`, `hasBela`. you can copy pretty much exactly the `hasQt` and `hasQtWebEngine` primitives (the new primitive should also go in PyrPlatformPrimitives.cpp and not a new file) except for the name change.

Reply:
Perfect, will do for sure, I like hasBelaSupport